virt/virtual_machines/virtual_disks/virt-expanding-vm-disks.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: ASSEMBLY
+[id="virt-expanding-vm-disks"]
+= Expanding virtual machine disks
+include::_attributes/common-attributes.adoc[]
+:context: virt-expanding-vm-disks
+
+toc::[]
+
+You can increase the size of a virtual machine (VM) disk by expanding the persistent volume claim (PVC) of the disk.
+
+If your storage provider does not support volume expansion, you can expand the available virtual storage of a VM by adding blank data volumes.
+
+You cannot reduce the size of a VM disk.
+
+include::modules/virt-enlarging-vm-disk-pvc.adoc[leveloffset=+1]
+
+[id="additional-resources_expanding-disk-volume"]
+[role="_additional-resources"]
+=== Additional resources for volume expansion
```

Comment:
Both of the additional resources sections in this file should use the module-style `.Additional resources` formatting, rather than `===`. You also don't need the ID (but you do need the role).

```
[role="_additional-resources"]
.Additional resources
```

(https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections)

Reply:
Thanks! I can never remember 'additional resources' formatting because we did not use this heading when I was still a writer.